Tasks/powerShell/task.json

```diff
@@ -50,13 +64,25 @@
             "required":false,
             "helpMarkDown": "Current working directory when script is run.  Defaults to the folder where the script is located.",
             "groupName":"advanced"
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "defaultValue": "Write-Host 'Hello World'",
+            "required": true,
+            "helpMarkDown": "Write your scripts and have fun",
```

Comment:
Can you check with PM and make the English similar to how other helpmarkdown is.